Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/ScanAPIModelWizardPage.xaml.cs

```diff
@@ -493,5 +501,24 @@ private void XManualMatchBtn_Click(object sender, RoutedEventArgs e)
             }
 
         }
+
+        private void xClearMatchBtn_Click(object sender, RoutedEventArgs e)
+        {
+            DeltaAPIModel deltaAPI = null;
+            deltaAPI = GetFrameElementDataContext(sender) as DeltaAPIModel;
+            //DeltaAPIModel updatedDelta = Amdocs.Ginger.CoreNET.Application_Models.LearnAPIModelsUtils.CreateAPIModelObject(deltaAPI);
+            int modelIndex = xApisSelectionGrid.DataSourceList.IndexOf(deltaAPI);
+            if (deltaAPI != null)
+            {
+                if(deltaAPI.matchingAPIModel != null)
+                {
+                    ObservableList<ApplicationAPIModel> selectedMatchingAPIList = new ObservableList<ApplicationAPIModel>();
```

Comment:
didn't got why we do compare when user asking to Clear the match, please explain

Reply:
The PR was pulled mistakenly as work was incomplete at that point (yesterday).
I've pushed the changes today,